src/cascade/input_data/configuration/builder.py

```diff
@@ -0,0 +1,84 @@
+""" Functions for creating internal model representations of settings from EpiViv
+"""
+from cascade.model.grids import AgeTimeGrid, PriorGrid
+from cascade.model.rates import Smooth
+from cascade.input_data.configuration import ConfigurationError
+from cascade.core.context import ModelContext
+from cascade.dismod.db.metadata import IntegrandEnum
+
+RATE_TO_INTEGRAND = dict(
+    iota=IntegrandEnum.Sincidence,
+    rho=IntegrandEnum.remission,
+    chi=IntegrandEnum.mtexcess,
+    omega=IntegrandEnum.mtother,
+    prevalence=IntegrandEnum.prevalence,
+)
+
+MEASURE_ID_TO_RATE_NAME = {
+    6: "iota",  # Incidence
```

Comment:
Ah, you're right. There is another spot. It's in src/cascade/input_data/db/bundle.py but there I think it's mapping from measure_id to integrand name. I'm not going to fix this in this PR because I want to have a more holistic conversation about the relationship between GBD measures/rates/integrands in our code.